src/backend/blockchain/src/routes.py

```diff
@@ -0,0 +1,64 @@
+from src import app
+from .transaction import Transaction
+from .block import Block
+from .blockchain import blockchain
+from flask import request, jsonify
+
+
+@app.route("/")
+def index():
+    return "Hello Blockchain"
+
+
+@app.route('/chain', methods=['GET'])
+def get_chain():
+    chain = []
+    for block in blockchain.chain:
+        chain.append(block.toJSON())
+    return jsonify(length=len(chain), chain=chain), 200
+
+
+@app.route('/proof', methods=['POST'])
+def proof():
+    data = request.json
+    # proof = data["proof"]
+    # newBlock = Block()
+    # TODO: Get other block data from the request data, including the
+    # transaction amount and to/from wallet IDs
+    # Then, add/subtract the transaction amount from the respective walletIds
+    # Using blockchain.addToWallet and blockchain.subtractFromWallet
+    return "proof was hit. You sent:"+request.json
+
+
+@app.route('/wallet/addWallet', methods=['POST'])
+def addWallet():
+    try:
+        data = request.json
+        walletId = data["walletId"]
+        success = blockchain.add_wallet(walletId)
+        return jsonify(success=success), 201
+    except Exception as e:
+        return jsonify(err=str(e)), 400
```

Comment:
just to stay consistent, we should return "Exception" field in json instead of "err"

Reply:
I'd prefer to use `err` over `Exception`, just cause most APIs don't return `Exception` if they encounter the issue, they usually return `err` or `error`